docs/guidelines/components/tables.md

```diff
@@ -119,6 +123,34 @@ code_example]]
 code_example]]
 </div>
 
+**<div class="mgb-3">Basic table with multiple rows within a cell</div>**
+<div class="ez-guidelines-tables__with-header ez-guidelines-tables__with-multiple-rows-in-cell mgb-5">
+[[code_example {html}
+<div class="ez-table-header">
+    <div class="ez-table-header__headline">Workflow</div>
+</div>
+<table class="ez-table table">
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Limitations</th>
+            <th>Stages</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td class="ez-table__cell--top"><a href="#">Article Workflow</a></td>
```

Comment:
I don't think we have class `ez-table__cell--top` in our CSSes.

Reply:
True, it's part of `ezplatform-workflow`. Should we consider adding it or we better wait?

Reply:
`workflow` is a EE feature, this might be confusing for the open-source users that this class doesn't work. But on the other hand, we do not have documentation for the EE users.

Reply:
Updated based on discussion 👍